unabridged-works-volume-1/original/07-axe-to-the-root.adoc

```diff
@@ -913,16 +937,12 @@ Do not provoke the tender heart of the Lamb against you,
 who also hath the voice of a lion,
 and can roar terribly out of his holy mountain against the enemies of his life and Spirit.
 
-=== A NECESSARY WARNING
-
-And of very great Importance to all that call themselves CHRISTIANS,
-and hope for a Share in the Book of LIFE, and the escaping the Damnation of Hell;
-which is their Portion whose Names are written in the Book of Death,
-and blotted by God out of the Book of LIFE, though they hope to find them written there.
+[.old-style]
+=== A Necessary Warning / And of very great Importance to all that call themselves CHRISTIANS, and hope for a Share in the Book of LIFE, and the escaping the Damnation of Hell; which is their Portion whose Names are written in the Book of Death, and blotted by God out of the Book of LIFE, though they hope to find them written there.
 
-HEAR AND CONSIDER
+==== Hear and Consider
```

Comment:
I'm not sure if this is the best way to have it left justified?

Reply:
I'm not sure this is even in the styleguide, but you can add a class `[.no-indent]` if you just want to eliminate the indentation.  But that class doesn't give you an extra space below, so we could use `[.offset]` to do both.

But... I sort of like the way you did it, so let's just leave it.